docs/DeveloperGuide.md

```diff
@@ -728,26 +728,82 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 Given below are instructions to test the app manually.
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
-testers are expected to do more *exploratory* testing.
+testers are expected to do more *exploratory* testing. Alternative test cases with similar expected results are provided but are **not exhaustive**.
 
 </div>
 
 ### Launch and shutdown
 
-1. Initial launch
+#### Initial launch
 
-   1. Download the jar file and copy into an empty folder.
+**Prerequisites:**
+1. `Java 11` is installed in the system.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+**Steps:**
+1. Download the latest `findvisor.jar` file and move it into an empty folder.
+2. Open the command terminal at the folder containing `findvisor.jar`.
+3. Run `java -jar FINDvisor.jar`.
 
-1. Saving window preferences
+**Expected Result:**
+1. FINDvisor GUI appears in a minimized window.
+2. FINDvisor contains a list of sample contacts.
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+#### Saving workspace settings
 
-   1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+**Steps:**
+1. Test case: Resize the window to a preferred size.
+2. Close FINDvisor.
+3. Relaunch FINDvisor through the command terminal as stated in [initial launch](#initial-launch).
 
-1. _{ more test cases …​ }_
+**Expected Result:**
+1. FINDvisor retains the previous window state before it was closed.
+
+**Alternative Test Cases:**
+- Shifting the window to a desired position.
+- Maximizing the window and restoring the window from maximized state.
+- Changing the divider position between *Today's Meeting Panel* and other UI components.
+
+### Adding a person
+
+#### Adding a person successfully
+
+**Prerequisites:**
+1. There is no person in the list with the same mobile phone number as the person to be added.
+
+**Steps:**
```

Comment:
For these single-line test cases, would it be better to rename this to `Example test case:` instead of `Steps:`?

Reply:
I am more inclined to keep `Steps :` for consistency of multiline steps although I think it is possible to change it. What do the rest think @Arixeyeion @Dethada @YuZizhen ?